StackOverflow/Controller/QuestionsTableViewController.m

```diff
@@ -0,0 +1,97 @@
+//
+//  QuestionsTableViewController.m
+//  StackOverflow
+//
+//  Created by Mali on 2017/10/04.
+//  Copyright © 2017 DVT. All rights reserved.
+//
+
+#import "QuestionsTableViewController.h"
+#import "QuestionsCell.h"
+#import "DataFetchDelegate.h"
+#import "QuestionsDataObject.h"
+#import "QuestionsData.h"
+
+
+@interface QuestionsTableViewController () <DataFetchDelegate> {
+    NSArray *dataResult;
+    QuestionsDataObject *dataObject;
+    UIActivityIndicatorView *activityIndicator;
+}
+
+@end
+
+@implementation QuestionsTableViewController
+
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    //Display a code loading indicator
+    activityIndicator = [[UIActivityIndicatorView alloc] initWithActivityIndicatorStyle:UIActivityIndicatorViewStyleGray];
+    activityIndicator.alpha = 1.0;
+    [self.view addSubview:activityIndicator];
+    activityIndicator.center = CGPointMake([[UIScreen mainScreen]bounds].size.width/2, [[UIScreen mainScreen]bounds].size.height/2);
```

Comment:
Views should be positioned using Auto Layout not by directly setting their location.